ncm-spma/src/main/perl/spma/dnf.pm

```diff
@@ -125,7 +125,15 @@ sub generate_dnf_modules {
         my $fh = $trd->filewriter("$directory/$name.yaml",
                                   header => "# File generated by " . __PACKAGE__ . ". Do not edit",
                                   log => $self);
-        $changes += $fh->close() || 0; # handle undef
+        $module_changed += $fh->close() || 0; #handle undef
+        $changes += $module_changed; 
+        if ($module_changed > 0 and !$CAF::Object::NoAction) {
+            my ($cmd_exit, $cmd_out, $cmd_err) = $self->execute_command([DNF_MODULE_RESET, $name], "module stream change detected. Reset managed module stream ".$name." for update", 1, "/dev/null", 1);
+            if ($cmd_exit) {
+                $self->error("Error during reset module ". $cmd_err);
```

Comment:
Use "Error during reset module $cmd_err"

The only reason to use concat (.) is if you need to use ' instead of "

Reply:
Thanks. updated